### ISSUE TITLE
Maccy and Word

### ISSUE THREAD

**Issue by Pre-Cambrian** [p0deje/Maccy#353]
Thanks for your quick response to my review.  I'm using Word Version 16.56 with Mac OS 12.1.  What I copy from Word is added to the clipboard but it will only paste the last item copied.  Maccy works with all the other apps I've tried it with, including Excel, and items copied from Excel to the clipboard can be pasted into Word, so it does seem to be a Word problem.

**Comment by p0deje** [comment]
This is exactly the issue I was trying to fix in https://github.com/p0deje/Maccy/commit/58000e00e1a5461b3a14ececd6b3500e580dd1ae, I'll need to double-check if Word 16 adds something else to copied content.

**Comment by p0deje** [comment]
I just checked Word 16.42 (it says it's the latest for me) with macOS 12.0.1 and I can paste any text from Maccy without any problems. Could you please help me debug the issue by installing https://apps.apple.com/app/id1499215709 (it's free), then copying something from Word and sharing a screenshot of this app? You should see something like this:
<img width="1840" alt="Screen Shot 2022-01-10 at 13 26 53" src="https://user-images.githubusercontent.com/665846/148730707-0052e2b5-da48-4a02-b26d-96dcc9e83f38.png">

**Comment by Pre-Cambrian** [comment]
Is this what you need to see?  I sent an email but it may not have got through.

![Screenshot 2022-01-21 at 18 32 03](https://user-images.githubusercontent.com/97389695/150581440-6e42766a-feb2-4617-b49a-4bdf876f0aee.jpeg)

**Comment by p0deje** [comment]
Yes, this is exactly what I was looking for!  Can I ask for a couple more things?

- Scroll the list on the left sidebar to show all the pasteboard types. 
- Copy something from Word, then copy something else, then select the first item in Maccy and take screenshot of Pasteboard Viewer app again.

**Comment by Pre-Cambrian** [comment]
Hoping I've followed the instructions correctly.
<img width="2048" alt="Screenshot 2022-01-22 at 12 07 06" src="https://user-images.githubusercontent.com/97389695/150637940-229f903b-ba4e-4327-a0f7-66112e04fbdb.png">

**Comment by Pre-Cambrian** [comment]
Do you have an update on this one?  Thanks.

**Comment by p0deje** [comment]
@Pre-Cambrian Sorry but I'm still unable to reproduce the problem. Can you please confirm that the screenshot above is what you did when I asked to "Copy something from Word, then copy something else, then select the first item in Maccy and take screenshot of Pasteboard Viewer app again."?

**Comment by p0deje** [comment]
@Pre-Cambrian I still cannot reproduce the problem. Here is the video of the it working fine for me. Can you please record something similar?

https://user-images.githubusercontent.com/665846/152630301-d43d85d4-b469-409b-9533-a99314d666e2.mov

**Comment by Pre-Cambrian** [comment]
https://user-images.githubusercontent.com/97389695/153431131-b41f5302-db2e-424d-91a4-a7d12508e195.mov

**Comment by p0deje** [comment]
@Pre-Cambrian Thank you, it turned out that I should have tested on the App Store version of Word. The issue is now fixed so please upgrade to Maccy 0.22.2 and let me know if it still happens. If it works fine now, I'd appreciate if you update your App Store review to note that the issue is fixed.
